File: GoChat/src/main/java/com/jw/gochat/receiver/PushReceiver.java
package com.jw.gochat.receiver;

import android.content.BroadcastReceiver;


public abstract class PushReceiver extends BroadcastReceiver {

	public final static String ACTION_TEXT = "action.text";
	public final static String ACTION_INVATION = "action.invation";
	public final static String ACTION_REINVATION = "action.reinvation";
	public final static String ACTION_ICON_CHANGE = "action.iconchange";
	public final static String ACTION_NAME_CHANGE = "action.namechange";

	public final static String KEY_FROM = "from";
	public final static String KEY_TO = "to";

	public final static String KEY_TEXT_CONTENT = "text_content";
}

File: goChat/src/main/java/com/jw/gochat/receiver/PushReceiver.kt
package com.jw.gochat.receiver

import android.content.BroadcastReceiver

abstract class PushReceiver : BroadcastReceiver() {
    companion object {
        val ACTION_TEXT = "action.text"
        val ACTION_INVATION = "action.invation"
        val ACTION_REINVATION = "action.reinvation"
        val ACTION_ICON_CHANGE = "action.iconchange"
        val ACTION_NAME_CHANGE = "action.namechange"
        val KEY_FROM = "from"
        val KEY_TO = "to"
        val KEY_TEXT_CONTENT = "text_content"
    }
}